src/test/java/io/specto/hoverfly/ruletest/HoverflyRuleRemoteInstanceTest.java

```diff
@@ -30,10 +29,10 @@
     @ClassRule
     public static HoverflyRule hoverflyRule = HoverflyRule.inSimulationMode(classpath("test-service-https.json"),
```

Comment:
Are we actually starting a remote Hoverfly yet (locally pretending to be remote)